docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+
+The set of channel members who need to approve a chaincode before it can be deployed is governed by the `Application/Channel/lifeycleEndorsement` policy. By default, this policy requires that a majority of channel members need to approve a chaincode before it can used on a channel. Because we have only two organizations on the channel, and a majority of 2 is 2, we need approve a chaincode definition of Fabcar as Org1 and Org2.
+
+If an organization installed the chaincode, they need to include the packageID in the chaincode definition approved by their organization. The package ID is used to associate the chaincode installed on a peer with an approved chaincode definition. You can find the package ID of a chaincode by using the `peer lifecycle chaincode queryinstalled` command to query your peer.
+```
+peer lifecycle chaincode queryinstalled
+```
+
+The package ID is the combination of the chaincode label and a hash of the chaincode bytes. You should see output similar to the following:
+```
+Get installed chaincodes on peer:
+Package ID: fabcar_1:69de748301770f6ef64b42aa6bb6cb291df20aa39542c3ef94008615704007f3, Label: fabcar_1
+```
+
+We are going to use the package ID when we approve the chaincode, so let's go ahead and save it as an environment variable. Paste the package ID returned by `peer lifecycle chaincode queryinstalled` into the command below. The package ID may not be the same for all users, so you need to complete this step using the package ID returned from your command window.
+```
+CC_PACKAGE_ID=fabcar_1:69de748301770f6ef64b42aa6bb6cb291df20aa39542c3ef94008615704007f3
+```
+
+Because you set the environment variables to operate the `peer` CLI as the Org2 admin, we can approve the chaincode definition of Fabcar for Org2. Chaincode is approved at the organization level, so the command only needs to target one peer. The approval is distributed to peers within each organization using gossip.
+```
+peer lifecycle chaincode approveformyorg -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name fabcar --version 1.0 --init-required --package-id $CC_PACKAGE_ID --sequence 1 --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+
+The command above uses the `--package-id` flag to include the package identifier in the chaincode definition. The command also uses the `--init-required` flag to request that Init function be invoked initialize the chaincode, before other functions can be used to interact with the ledger. By default, the Init function is not required and cannot be executed. You do not need to request that the Init function be invoked to initialize your chaincode.
```

Comment:
I was a little confused when you set --init-required but you also say 
By default, the Init function is not required and _cannot be executed._
If it cannot be executed, why are we setting?

Reply:
Changing By default, the Init function is not required and cannot be executed. > By default, the Init function does not need to be executed.

The Init function is required, but you do not need to request that it be invoked (I know, I know)